app/views/account_messenger/expired_credit_card_notification_for_buyer.text.liquid

```diff
@@ -4,8 +4,8 @@ Thank you for using our service.
 
 We are writing you because your Credit Card is about to expire. To continue using the service, please update the new payment details in the next 10 days. You can find them on the Account tab of your API Dashboard.
 
-{% if admin_support_email %}
-If you need any help, please don't hesitate to contact us on {{admin_support_email}}.
+{% if support_email %}
```

Comment:
`admin_support_email` has been replaced for `support_email`, that's good. I wonder if we still need the `if support_email` or if it is always present 🤔

Reply:
I've kept it just in case, although it seems that it's not possible to set `support_email` to a null value (at least when trying through the UI `/site/emails`).

Reply:
Ok, it can be empty 😄 
https://github.com/3scale/porta/blob/245555bbef2f265cec806a8a6f40152a1ce57e66/app/models/service.rb#L23-L24
https://github.com/3scale/porta/blob/245555bbef2f265cec806a8a6f40152a1ce57e66/db/schema.rb#L87